test/dev/gethop/payments/stripe/promotion_code_test.clj

```diff
@@ -0,0 +1,65 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+(ns dev.gethop.payments.stripe.promotion-code-test
+  (:require [clojure.string :as str]
+            [clojure.test :refer :all]
+            [dev.gethop.payments.core :as core]
+            [dev.gethop.payments.stripe]
+            [dev.gethop.payments.stripe.test-utils :as test-utils]))
+
+(deftest ^:integration create-promotion-code
+  (let [payment-adapter (test-utils/init-payment-adapter)]
+    (testing "Create valid promotion code"
+      (let [coupon-id (test-utils/create-coupon+get-id payment-adapter)
```

Comment:
Unless I'm missing something, I don't see an implementation for test-utils/create-coupon+get-id

Reply:
I missed dev.gethop.payments.stripe.test-utils in the commit, sorry!

Reply:
changed